WireAuthentication/Sources/WireAuthenticationUI/Views/DetermineAuthMethod/DetermineAuthMethodViewModel.swift

```diff
@@ -102,15 +107,22 @@ package final class DetermineAuthMethodViewModel: ObservableObject {
     func didDismissAlert(alert: Alert) {
         switch alert {
         case let .onPremLoginNotPossible(method):
-            handleAuthenticationMethod(method)
+            Task.detached {
```

Comment:
question: why detached?

Reply:
because handleAuthenticationMethod was async, but I changed it